server/serializers/subscriptions.go

```diff
@@ -20,12 +20,18 @@ type PublisherInputsGeneric struct {
 
 type PublisherInputsBoards struct {
 	ProjectID string `json:"projectId"`
+	AreaPath  string `json:"areaPath,omitempty"`
 }
 
 type PublisherInputsRepos struct {
-	ProjectID  string `json:"projectId"`
-	Repository string `json:"repository"`
-	Branch     string `json:"branch"`
+	ProjectID                    string `json:"projectId"`
+	Repository                   string `json:"repository,omitempty"`
+	Branch                       string `json:"branch,omitempty"`
+	PullRequestCreatedBy         string `json:"pullrequestCreatedBy,omitempty"`
+	PullRequestReviewersContains string `json:"pullrequestReviewersContains,omitempty"`
```

Comment:
```suggestion
	PullRequestCreatedBy         string `json:"pullRequestCreatedBy,omitempty"`
	PullRequestReviewersContains string `json:"pullRequestReviewersContains,omitempty"`
```

Reply:
these cannot be modified as these are the expected key names for AZD